controlpanel/settings/common.py

```diff
@@ -477,7 +486,7 @@
     CACHES = {
         "default": {
             "BACKEND": "django_redis.cache.RedisCache",
-            "LOCATION": f"redis://{REDIS_HOST}:{REDIS_PORT}/1",
```

Comment:
Aha.... found the "`1`". Still not sure of its significance or (apparent) difference in use from before (see comments above).

Reply:
I'm not massively familiar with Redis neither but that number after the "slash" is the DB it will use: https://stackoverflow.com/a/20770414/455642

It seemed to work fine in `dev`, both login/session and background jobs submission/execution. I suspect this may be more useful when you share a Redis cluster among several applications or something like that.

Reply:
Ahh... yes. I know about REDIS database numbers, but didn't realise this was the reference to it. ;-) :+1:

Reply:
Me neither, I only found out while refactoring this.

In theory unless both Django and Django Channels use the same keys it should be a problem for them to use the same DB/keyspace and it doesn't seem to be a problem in `dev`, but do you think we should use two DBs here?